beanie/odm/interfaces/find.py

```diff
@@ -27,10 +27,15 @@
 
 if TYPE_CHECKING:
     from beanie.odm.documents import Document
+    from beanie.odm.union_doc import UnionDoc
     from beanie.odm.views import View
 
+    FindType = TypeVar("FindType", bound=Union["Document", "UnionDoc", "View"])
+else:
+    FindType = TypeVar("FindType", bound=Union["Document", "View"])
```

Comment:
Please correct me if I'm wrong, but aren't both of these types intended for type checking?